pyproject.toml

```diff
@@ -49,7 +49,8 @@ dependencies = [
   'typing-extensions~=4.0;python_version<"3.10"',
   'upf_to_json~=0.9.2',
   'wrapt~=1.11',
-  'chardet~=5.2.0;platform_system=="Windows"'
+  'chardet~=5.2.0;platform_system=="Windows"',
+  'keyringrs~=0.1.0'
```

Comment:
I release `0.3.1` which include the typing annotation and support for linux if only keyutils can be supported from the platform. Doesn't matter too much if this PR at the moment through.

Reply:
released https://pypi.org/project/keyringrs/